src/App/Pages/Settings/FolderAddEditPage.xaml

```diff
@@ -13,16 +13,17 @@
     </ContentPage.BindingContext>
 
     <ContentPage.ToolbarItems>
-        <ToolbarItem Text="{u:I18n Cancel}" Clicked="Close_Clicked" Order="Primary" Priority="-1"
-                     AutomationId="CancelButton" />
-        <ToolbarItem Text="{u:I18n Save}" Clicked="Save_Clicked" Order="Primary"
-                     AutomationId="SaveFolderButton" />
+        <ToolbarItem Text="{u:I18n Cancel}"
+                     Clicked="Close_Clicked"
+                     Order="Primary" Priority="-1" />
```

Comment:
`Priority` should be on its own line to match the new formatting

Reply:
Done!